kie-spring-boot/pom.xml

```diff
@@ -34,10 +34,9 @@
 
   <properties>
     <version.org.apache.cxf>3.2.6</version.org.apache.cxf>
```

Comment:
arent we syncing with bootstrap pom ?

Reply:
good question, there should be at least the same version like in kie-parent 3.2.12, not sure about SB one, but the previous version of this PR having 3.2.12 failed in tests for cxf integrations.

Reply:
@AntStephenson possible needed for https://issues.redhat.com/browse/RHPAM-2734